src/lib_ccx/lib_ccx.h

```diff
@@ -250,7 +250,7 @@ int myth_loop(struct lib_ccx_ctx *ctx);
 int matroska_loop(struct lib_ccx_ctx *ctx);
 
 // utility.c
-void fatal(int exit_code, const char *fmt, ...);
+void fatal(int exit_code, const char *fmt, ...) __attribute__ ((noreturn));
```

Comment:
That's not portable - remember, we're not just using GCC, we also use Visual Studio for the Windows builds!

Reply:
I was internally dreaming MSVC had the same extension.

Would you be okay with some pre processor stuff to use the equivalent MSVC extenison?

https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2012/dabb5z75(v=vs.110)?redirectedfrom=MSDN

Reply:
There also `_Noreturn` which is in the C11 standard